include/kansei/imu.hpp

```diff
@@ -46,9 +45,9 @@ class Imu
 
   void compute_rpy(float gy[3], float acc[3], float seconds);
 
-  float get_roll() {return keisan::rad_to_deg(roll);}
-  float get_pitch() {return keisan::rad_to_deg(pitch);}
-  float get_yaw() {return keisan::rad_to_deg(yaw);}
+  float get_roll() {return roll;}
+  float get_pitch() {return pitch;}
+  float get_yaw() {return yaw;}
```

Comment:
Use `const type & function_name() const` for a getter that return a member variable of a class. Do this to the other function too.